site/docs/guides/llm-redteaming.md

```diff
@@ -48,6 +48,14 @@ prompts:
 
 providers:
   - openai:gpt-3.5-turbo
+
+redteam:
```

Comment:
I would remove this from the quickstart, or put it into a `tip` that links to the relevant configuration section